recipes/grpc/all/conanfile.py

```diff
@@ -0,0 +1,161 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+from conans.tools import Version
+import os
+
+
+class grpcConan(ConanFile):
+    name = "grpc"
+    description = "Google's RPC library and framework."
+    topics = ("conan", "grpc", "rpc")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/grpc/grpc"
+    license = "Apache-2.0"
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake", "cmake_find_package_multi"
+    short_paths = True
+
+    settings = "os", "arch", "compiler", "build_type"
+    # TODO: Add shared option
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "build_codegen": [True, False],
+        "build_csharp_ext": [True, False],
+        "build_cpp_plugin": [True, False],
+        "build_csharp_plugin": [True, False],
+        "build_node_plugin": [True, False],
+        "build_objective_c_plugin": [True, False],
+        "build_php_plugin": [True, False],
+        "build_python_plugin": [True, False],
+        "build_ruby_plugin": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "build_codegen": True,
+        "build_csharp_ext": False,
+        "build_cpp_plugin": True,
+        "build_csharp_plugin": True,
+        "build_node_plugin": True,
+        "build_objective_c_plugin": True,
+        "build_php_plugin": True,
+        "build_python_plugin": True,
+        "build_ruby_plugin": True,
+    }
+
+    _source_subfolder = "source_subfolder"
+    _build_subfolder = "build_subfolder"
+
+    requires = (
```

Comment:
Are those versions in sync with GRPC 1.37.0?

I learned that grpc is pretty picky about getting exact versions of its dependencies

Reply:
At least it compiles... but I didn't check